docs/jina-ai-cloud/login.md

```diff
@@ -76,19 +76,10 @@ Installed along with Jina, you can leverage the `hubble` package to manage login
 ```python
 import hubble
 
-# Open browser automatically and login via 3rd party.
-# Token will be saved locally.
-hubble.login()
-```
-
-#### Login from Jupyter notebook/Google Colab
-
-```python
-import hubble
-
-# Use Personal Access Token or browser to login.
-# Token will be saved locally.
-hubble.notebook_login()
+# Login via browser or Personal Access Token. The token will be saved locally.
```

Comment:
Change "login" to "log in" since it's used as a verb ("login" is for noun/adjective)

Reply:
Apply throughout the doc